src/main/java/org/apache/cassandra/sidecar/utils/SSTableImporter.java

```diff
@@ -310,6 +307,31 @@ private void cleanup(ImportOptions options)
                                                  cause));
     }
 
+    /**
+     * Aggregates pending imports per host from multiple keyspaces and tables
+     */
+    private void reportPendingImportAggregates()
```

Comment:
nit: the method name is a bit confusing with "Aggregates". How about `reportPendingImportPerHost`?